cmd/oracled/cmd/update_oracle_info.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/medibloc/panacea-oracle/client/flags"
+	"github.com/medibloc/panacea-oracle/panacea"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func updateOracleInfoCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "update-oracle-info",
+		Short: "Update an oracle's info",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			conf, err := loadConfigFromHome(cmd)
+			if err != nil {
+				return err
+			}
+
+			// get trusted block information
+			trustedBlockInfo, err := getTrustedBlockInfo(cmd)
+			if err != nil {
+				return fmt.Errorf("failed to get trusted block info: %w", err)
+			}
+
+			queryClient, err := panacea.NewVerifiedQueryClient(context.Background(), conf, *trustedBlockInfo)
```

Comment:
There is no need to receive trusted block info, actually must not.
Let's load verified query client using conf.
```go
queryClient, err := panacea.LoadVerifiedQueryClient(context.Background(), conf)
```

Reply:
Actually, I missed it 😂 Thx!